EN_README.md

```diff
@@ -0,0 +1,62 @@
+# Study Reminder
+
+This project is a simple application developed in Python using the Tkinter library. The application presents a reminder with a personalized message and allows the user to confirm a specific action, which is executed via a predefined system command.
```

Comment:
Let's be a little bit more human about this explanation. Something like:
"The main goal of this app is to prevent opening apps from your desktop without thinking about your personal commitements first. This app helps you with that adding some friction when you try to open games, browsers or any other app that distracts you from your goals..."